setup.py

```diff
@@ -62,6 +62,7 @@
         "torcheval~=0.0.6",
         "tqdm~=4.62",
         "typing_extensions~=4.12",
+        "blobfile~=3.0.0",  # This dependency is required for tiktoken.load.read_file, but it's listed as optional in tiktoken's pyproject.toml (https://github.com/openai/tiktoken/blob/main/pyproject.toml#L9)  # fmt: skip
```

Comment:
Could you please move this inline comment on top of line 65 and limit the comment line length to 80 chars?

Reply:
Done with the new commit!